courses/machine_learning/deepdive2/launching_into_ml/solutions/python.BQ_explore_data.ipynb

```diff
@@ -41,6 +41,7 @@
    "metadata": {},
```

Comment:
<p>Line 1, "value".  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1266/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Updated the change.